### PR TITLE
Update logos of Products

### LINKED ISSUE #360: Update the logos of products
We need to update the following product's logos:
1. Open Source
2. Chooser
3. Search
4. Global Network
5. Global Summit
6. State of the Commons
7. Certificates
8. Vocabulary

The SVG files can be exported from [the Design Library](https://www.figma.com/file/l4Mt3dn3Ndtrvrb4aLcwXI/Design-Library?node-id=784%3A0&viewport=252%2C-29%2C0.8688915371894836)

### PR DESCRIPTION
**Pull request by kushaggarwal**
## Fixes

Fixes #360 by @panchovm 

## Description

This PR aims at the updation of the current svg for various CC products

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by kushaggarwal** [comment]
Hello @dhruvkb 
I have updated all the product logos for the products mentioned in the figma design library
As per the old products there are certain other styles in which those were present earlier like certificates colorful
So, shall I remove all the related mdx stories from the current file
Thank you

**Comment by dhruvkb** [comment]
Yes @kushaggarwal, remove the stories that have become irrelevant.

**Comment by kushaggarwal** [comment]
Hello @dhruvkb 
Please find the updated logos for CC Products in the above linked pull request
Thank you

File: src/assets/logos/products/state_of_the_commons.svg [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<path d="M379.655 23.254C379.655 19.956 377.207 18.528 374.385 17.916L372.039 17.406C371.461 17.27 370.169 17.1 369.659 16.76C369.387 16.556 369.183 16.25 369.183 15.91C369.183 15.604 369.353 15.162 369.999 14.856C370.237 14.754 370.917 14.55 371.665 14.55C372.787 14.55 374.215 15.06 375.167 15.638C375.507 15.842 375.813 16.08 376.119 16.318L379.349 13.326C378.873 12.986 378.397 12.68 377.887 12.374C375.201 10.844 372.583 10.674 371.461 10.674C367.517 10.674 365.511 12.51 364.695 13.972C364.049 15.094 363.947 16.148 363.947 16.624C363.947 17.032 364.015 17.882 364.491 18.834C365.409 20.602 367.109 21.452 370.815 22.064C372.719 22.37 374.283 22.506 374.317 23.73C374.317 24.716 373.263 25.158 372.481 25.328C371.903 25.464 371.427 25.498 371.189 25.498C369.727 25.498 368.129 24.818 366.973 23.9C366.633 23.662 366.327 23.356 366.021 23.05L362.689 26.042C363.029 26.348 363.743 27.062 364.797 27.708C367.381 29.306 370.339 29.476 371.291 29.476C373.977 29.476 377.241 28.558 378.805 26.212C379.111 25.77 379.655 24.716 379.655 23.254Z" fill="black"/>
+</g>
```
Comment (dhruvkb) [inline src/assets/logos/products/state_of_the_commons.svg]: @kushaggarwal a little bit of formatting such as indentation would help a lot here.

File: src/stories/logos.stories.mdx [hunk 2]
```diff
@@ -18,6 +18,8 @@ export const blackText = (content) => `<div class="has-text-black">${content}</d
```
Comment (dhruvkb) [inline src/stories/logos.stories.mdx]: Please remove inverted versions of lettermark and letterheart.

Comment (dhruvkb) [reply src/stories/logos.stories.mdx]: Use the `plainText` function for them as well.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Some very minor changes and this is good to go.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Also update the code pertaining to the CC Legal Database logo.

**Comment by kushaggarwal** [comment]
Okay @dhruvkb 
Will update the following

**Comment by kushaggarwal** [comment]
Hello @dhruvkb 
Please find the above changes related to CC Legal, Lettermark and Letterheart
Thanks

**Review by dhruvkb — APPROVED** [review]

**Pull request merged**